src/components/SchemaRow/useChoices.ts

```diff
@@ -18,7 +18,7 @@ function calculateChoiceTitle(node: SchemaNode, isPlural: boolean): string {
     if (realName) {
       return realName;
     }
-    return node.primaryType !== null ? node.primaryType + primitiveSuffix : 'any';
+    return node.primaryType !== null ? node.primaryType + primitiveSuffix : node.originalFragment.title as string || 'any';
```

Comment:
```suggestion
    return node.primaryType !== null ? node.primaryType + primitiveSuffix : String(node.originalFragment.title || 'any');
```

That's to cover for any potentially invalid json schema.